examples/platform/silabs/SiWx917/uart.cpp

```diff
@@ -181,7 +180,10 @@ int16_t uartConsoleRead(char * Buf, uint16_t NbBytesToRead)
     {
         return UART_CONSOLE_ERR;
     }
-    status = UARTdrv->Receive(Buf, NbBytesToRead);
+    // storing the contents of rx_buffer in the Buf
+    *Buf = (char) rx_buffer;
```

Comment:
So this reads only one byte, right?  But the caller asked for NbBytesToRead bytes...

Reply:
fixed the flow on the latest commit